### PR TITLE
[full-ci] [tests-only] choose-tests-for-ocisSmokeTest: add tag ocisSmokeTest

### PR DESCRIPTION
**Pull request by ScharfViktor**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISSharingInternalUsers2</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISSharingPermissions1</strong> failed. The build is cancelled...</summary>

File: tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature [hunk 1]
```diff
@@ -112,6 +113,7 @@ Feature: accept/decline shares coming from internal users
     Then file "toshare.txt" should not be listed on the webUI
     And file "anotherfile.txt" should not be listed on the webUI
 
+
```
Comment (phil-davis) [inline tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature:116]: This extra line can be removed.

**Review by phil-davis — COMMENTED** [review]
And please sort out whatever is needed to "sign the CLA"

File: tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature]: done

**Review by phil-davis — COMMENTED** [review]
@ScharfViktor we try to leave 2 lines between each scenario. That allows us to have an empty line, and then a line that has filter-tags on it. When there are no filter-tags for a scenario, that 2nd line is empty.

That allows us to add/change filter-tags on scenarios without the line numbers changing. And that reduces "annoying" changes to line-number references in expected-failures files.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISSharingPublic</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISSharingPermissions3</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIOCISBasic</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. The build is cancelled...</summary>

File: tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature [hunk 2]
```diff
@@ -25,7 +25,7 @@ Feature: files and folders can be deleted from the trashbin
     And user "Alice" has logged in using the webUI
     And the user has browsed to the trashbin page
 
-  @issue-1725
+  @issue-1725 @smokeTest @ocisSmokeTest @yetToImplement
```
Comment (phil-davis) [inline tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:28]:
```suggestion
  @issue-1725 @smokeTest @ocisSmokeTest
```

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/16816/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16829/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/16829/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by phil-davis — APPROVED** [review]
LGTM - please squash the commits

And it still thinks that you need to "sign"  the CLA.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16831/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp2</strong> https://drone.owncloud.com/owncloud/web/16831/63/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16833/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/16833/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by phil-davis — APPROVED** [review]

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16835/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/16835/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16837/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/16837/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by phil-davis** [comment]
@ScharfViktor you need to amend the first commit and make your real GitHub name and email address be the Author.

Maybe some git commands like this: https://www.codegrepper.com/code-examples/shell/how+to+remove+author+from+git+commit

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic</strong> https://drone.owncloud.com/owncloud/web/16847/6/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUILogin-adminBlocksUser-feature-20.png
  

### webUIPreview-imageMediaViewer-feature-141.png
  

### webUIPreview-imageMediaViewer-feature-159.png
  

### webUIPreview-imageMediaViewer-feature-84.png
  

### webUIPreview-imageMediaViewer-feature-91.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by phil-davis** [comment]
@ScharfViktor the problem was that the line numbers changed in some feature files, so the line numbers need to be adjusted in expected-failures files.

I push a commit for that. And I put `full-ci` in the PR title. That makes all the pipelines run to completion (no early fail). That will let us see all failures.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16850/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16857/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by phil-davis** [comment]
Now I understand the failures. Some scenarios have been added to the smoke tests, and pipelines like https://drone.owncloud.com/owncloud/web/16857/44/14 oC10iPhone2-chrome run just the smoke tests. So those pipelines now have new scenarios.

`webUITrashbinDelete/trashbinDelete.feature:29` is failing in that environment (smaller screen). It might be a real problem with the web app on a small screen, or it might be a problem with the test code not managing to scroll things into view correctly...

I raised issue #5372 and added the scenario to the relevant expected-failures files.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16858/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong> https://drone.owncloud.com/owncloud/web/16858/38/1
The following scenarios passed on retry:
- webUISharingExternalToRoot/federationSharing.feature:353

**Pull request merged**